src/App.css

```diff
@@ -1,3 +1,83 @@
-body {
+.body {
     background-color: gray;
-}
\ No newline at end of file
+
+}
+.app__container {
+    background-color: gray;
+    display: grid;
+    grid-template-rows: auto 1fr auto;
+    grid-template-columns: 25% 1fr;
+    height: 100%;
+  }
+  
+  .content__container {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: auto 1fr;
+    width: 100%;
+    height: auto;
+    margin-left: 1em;
+    margin-right: 1em;
+    overflow-y: auto;
+}
+
+.home-main__container {
+    height: 100%;
+}
+  
+.sidebar {
+    width: 100%;
+}
+  
+  .header {
+    background-color: #BADA55;
+  }
+  
+  .home-articles__container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: auto;
+    gap: 1rem;
+    background-color: blue;
+    margin-bottom: 3rem;
+
+  }
+  
+
+
+  @media only screen and (max-width: 999px) {
```

Comment:
Generally speaking, we should approach design mobile first - eg to swap the media query logic around here.
Have your default css apply to a small screen and alter it when getting larger